launcher/ui/themes/ThemeManager.cpp

```diff
@@ -36,6 +36,7 @@
 
 ThemeManager::ThemeManager()
 {
+    QIcon::setFallbackThemeName(QIcon::themeName());
```

Comment:
It's probably safer to also set the fallback path and set the main path to just :/icons